app/src/main/java/com/wafflestudio/snutt2/views/logged_in/home/HomeDrawer.kt

```diff
@@ -1,16 +1,52 @@
 package com.wafflestudio.snutt2.views.logged_in.home
 
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wafflestudio.snutt2.components.compose.clicks
+import com.wafflestudio.snutt2.lib.network.dto.core.CourseBookDto
+import com.wafflestudio.snutt2.lib.network.dto.core.SimpleTableDto
+import com.wafflestudio.snutt2.lib.toFormattedString
 
 @Composable
-fun HomeDrawer() {
-    Text(text = "drawer")
+fun HomeDrawer(
+    // TODO: CompositionalLocal 로 넘기는 형태로 바꾸기
+    selectedCourseBook: CourseBookDto,
+    selectedCourseBookTableList: List<SimpleTableDto>,
+    onClickItem: (String) -> Unit
```

Comment:
```suggestion
    onClickTableItem: (String) -> Unit
```

course book click 과 헷갈림을 방지하기 위해